reladomo/src/main/java/com/gs/fw/common/mithra/extractor/asm/ExtractorWriter.java

```diff
@@ -376,9 +376,21 @@ protected static ClassWriter createClassWriterWithConstructor(String className,
         return cw;
     }
 
-    public static String createClassName()
+//    private static void addToStringMethod(ClassWriter cw, String str)

```

Comment:
Did you mean to comment this out? The super class of these generated ones (Attribute) already has a toString.